jina/peapods/pods/k8s.py

```diff
@@ -127,14 +127,19 @@ def _deploy_runtime(
             replicas=replicas,
             pull_policy='IfNotPresent',
             init_container=init_container_args,
+            custom_resource_dir=getattr(self.args, 'k8s_custom_resource_dir', None),
         )
 
     def start(self) -> 'K8sPod':
         """Deploy the kubernetes pods via k8s Deployment and k8s Service.
 
         :return: self
         """
-        kubernetes_tools.create('namespace', {'name': self.args.k8s_namespace})
+        kubernetes_tools.create(
+            'namespace',
+            {'name': self.args.k8s_namespace},
+            custom_resource_dir=getattr(self.args, 'k8s_custom_resource_dir', None),
```

Comment:
default value should be set in parser

Reply:
Set in https://github.com/jina-ai/jina/pull/3309/commits/6f60fe45069236ba73d80bd5d07c514cf99dd229

Reply:
I use this logic to catch the case when the gateway is created. There the attribute does on exist on the args object.